R/database.R

```diff
@@ -65,7 +71,7 @@ get_ssm_by_patients = function(these_patient_ids,
     ungroup()
 
   these_sample_ids = pull(these_samples_metadata, sample_id)
-  return(get_ssm_by_samples(these_sample_ids, these_samples_metadata, tool_name, projection, seq_type, flavour, min_read_support, subset_from_merge))
+  return(get_ssm_by_samples(these_sample_ids, these_samples_metadata, tool_name, projection, seq_type, flavour, min_read_support, subset_from_merge, basic_columns, maf_cols, return_cols))
```

Comment:
It's probably safer to pass all these as named arguments in case the order or number of arguments in get_ssm_by_samples ever changes